contracts/VotableStakingRewards.sol

```diff
@@ -244,6 +213,31 @@ contract VotableStakingRewards is
     emit RewardsDurationUpdated(rewardsDuration);
   }
 
+  /* ========== VOTER FUNCTIONS ========== */
+    
+  function propose(
+    address[] memory targets,
+    uint256[] memory values,
+    string[] memory signatures,
+    bytes[] memory calldatas,
+    string memory description
+    ) external onlyOwner{
```

Comment:
Wouldn't this only be callable by the VotableStakingRewards owner? Try testing `propose` from a wallet that isn't the VotableStakingRewards owner.

I don't think you need any checks here. voters[msg.sender] already enforces that only the caller's voter is calling `propose`.